cli/configure/configure.go

```diff
@@ -252,16 +250,16 @@ func configureLocalCli(cmdCtx *cmdcontext.CmdCtx, launchDir string) error {
 	if localCli != currentCli {
 		if _, err := os.Stat(localCli); err == nil {
 			if _, err := exec.LookPath(localCli); err != nil {
-				return fmt.Errorf(
-					`Found tt binary in local directory "%s" isn't executable: %s`, launchDir, err)
-			}
-
-			// We are not using the "RunExec" function because we have no reason to have several
-			// "tt" processes. Moreover, it looks strange when we start "tt", which starts "tt",
-			// which starts tarantool or some external module.
-			err = syscall.Exec(localCli, append([]string{localCli}, os.Args[1:]...), os.Environ())
-			if err != nil {
-				return err
+				log.Warnf(`"%s" isn't executable: %s`, localCli, err)
```

Comment:
I don't understand 2 points:
- why is it bad to fail?
- If I understand correctly, before the changes, it should print an error that this is not an executable file, go fix it. Or am I missing something?